src/telegrafs/actions/thunks.ts

```diff
@@ -100,17 +109,19 @@ export const createTelegraf = (telegraf: Telegraf) => async (
 }
 
 export const updateTelegraf = (telegraf: Telegraf) => async (
-  dispatch: Dispatch<Action>,
-  getState: GetState
+  dispatch: Dispatch<Action>
 ) => {
   try {
-    const state = getState()
-    const labels = getLabels(state, telegraf.labels)
```

Comment:
Same as above, updating an existing telegraf config doesn't update labels, so no need to call `getLabels`.